src/Update.cpp

```diff
@@ -18,13 +18,30 @@ void RecordEvent(double, int, int, int, int); //added int as argument to InfectS
 
 Severity ChooseFinalDiseaseSeverity(int, int);
 
-// state transition helpers
+// infection state transition helpers
 void SusceptibleToRecovered(int cellIndex);
 void SusceptibleToLatent(int cellIndex);
 void LatentToInfectious(int cellIndex);
 void InfectiousToRecovered(int cellIndex);
 void InfectiousToDeath(int cellIndex);
 
+// severity state transition helpers
+
+void ToInfected(int tn, short infectType, int ai, double q);
+void FromMild(int tn, int microCellIndex, int ai);
+void ToMild(int tn, int microCellIndex, int ai);
+void FromCritRecov(int tn, int microCellIndex, int ai);
+void ToCritRecov(int tn, int microCellIndex, int ai);
+void FromSARI(int tn, int microCellIndex, int ai);
+void ToSARI(int tn, int microCellIndex, int ai);
+void FromILI(int tn, int microCellIndex, int ai);
+void ToILI(int tn, int microCellIndex, int ai);
+void FromCritical(int tn, int microCellIndex, int ai);
+void ToCritical(int tn, int microCellIndex, int ai);
+void ToDeathILI(int tn, int microCellIndex, int ai);
+void ToDeathSARI(int tn, int microCellIndex, int ai);
+void ToDeathCritical(int tn, int microCellIndex, int ai);
+
```

Comment:
If only used in this file please use `static` or an anonymous namespace.

Reply:
helper functions are now static